### ISSUE TITLE
tflint in docker - can't see modules

### ISSUE THREAD

**Issue by juldou** [wata727/tflint#206]
Problem is that terraform is working differently on tflint in docker and installed with brew.

With brew, it works just fine - I run terraform get and after tflint - code is ok.

But, when is it ran with docker: following happens: `docker run --rm -v $(pwd):/data  -t wata727/tflint`
the output is: `ERROR: module `./vpc` not found. Did you run `terraform get`?`

It seems to me that docker can't handle .terraform file inside container.  (but I'm not sure)

**Comment by wata727** [comment]
Thanks for your report.
Umm... I can't reproduce this problem.

TFLint checks `.terraform/modules` directory for modules. Perhaps the `.terraform` directory is not mounted in a container. Please check the output of the following command.

```
$ docker run --rm -v $(pwd):/data --entrypoint ls -t wata727/tflint -la
total 60
drwxr-xr-x   10 root     root           320 Jan 30 13:38 .
drwxr-xr-x    1 root     root          4096 Jun 19 12:38 ..
drwxr-xr-x    4 root     root           128 Jan 30 13:29 .terraform
-rw-r--r--    1 root     root            76 Jan 28 06:18 Gemfile
-rw-r--r--    1 root     root           400 Jan 28 06:21 Gemfile.lock
-rw-r--r--    1 root     root           383 Jan 28 06:27 template.tf
-rw-r--r--    1 root     root           498 Jan 28 06:23 template.tf.erb
-rw-r--r--    1 root     root           519 Jan 30 13:38 terraform.tfstate
-rw-r--r--    1 root     root         29762 Jan 30 13:35 terraform.tfstate.backup
```

The above is my example. TFLint doesn't raise an error because the `.terraform` directory exists.

**Comment by dr-jose** [comment]
The issue seems to be that the `.terraform` directory preserves links to the file system which don't exist on the mounted volume.  Here's an example using tflint version `0.7.0` and  terraform `0.11.7`

`/data # tflint --aws-region=us-east-1 --error-with-issues --deep
ERROR: module `environments/us-east-1` not found. Did you run `terraform get`?
/data # cd .terraform/
/data/.terraform # ls -ltra modules/
total 4
lrwxr-xr-x    1 root     root            60 Jul  6 16:42 bba042c568805c977510b39540240c0d -> /Users/exx914/work/trf/infrastructure/environments/us-east-1
lrwxr-xr-x    1 root     root            44 Jul  6 16:42 5fdfc6baf6c4a7934068ecc5b4c99d5c -> /Users/exx914/work/trf/infrastructure/stacks
lrwxr-xr-x    1 root     root            65 Jul  6 16:42 18240f9cde5d2c9277f455785620ab37 -> /Users/exx914/work/trf/infrastructure/environments/us-east-1/beta
drwxr-xr-x    3 root     root            96 Jul  6 16:42 ..
drwxr-xr-x    5 root     root           160 Jul  6 16:42 b0a29155b95fa0b04acccea5f1e6a934
lrwxr-xr-x    1 root     root            48 Jul  6 16:42 a06871b570150f7818c84d81757d1bb5 -> /Users/exx914/work/trf/infrastructure/stacks/k8s
-rw-r--r--    1 root     root          2077 Jul  6 16:42 modules.json
lrwxr-xr-x    1 root     root            89 Jul  6 16:42 609479266d9d872dbbf8693145a815c8 -> /Users/exx914/work/trf/infrastructure/.terraform/modules/3ccf1704bebf754877377af96046cfde
drwxr-xr-x    7 root     root           224 Jul  6 16:42 3ccf1704bebf754877377af96046cfde
lrwxr-xr-x    1 root     root            60 Jul  6 16:42 289d19a64f13d731bb715740d5a67b61 -> /Users/exx914/work/trf/infrastructure/modules/aws/kubernetes
drwxr-xr-x   11 root     root           352 Jul  6 16:42 .
`

**Comment by wata727** [comment]
Hi there, I just released TFLint v0.8.0 that supports v0.11 and v0.12 modules. Please try the latest version again. If you still have problems, please open a new issue instead of reopening.
